lab-anthony/test/storage-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fs = require('fs');
+
+const Storage = require('../lib/storage');
+const dataDir = new Storage(`${__dirname}/../data`);
+
+describe('Testing the storage module', function() {
+  describe('Testing successful setItem function', function(){
+    it('should write a new file by item id ', function(done){
+      const item = {id:8888};
+      dataDir.setItem('notes', item).then(function(data){
+        expect(data).to.be.equal(item);
+        expect(data.id).to.be.equal(8888);
```

Comment:
Super nit-picky, but the whole point of the filler words is to make your tests read like sentences. If you were going to say this, you would say 
```
I expect this to equal that
```
or something along those lines, so don't add 'be' fillers where they aren't helpful.